common/p_interaction.cpp

```diff
@@ -1031,6 +1034,55 @@ void SexMessage (const char *from, char *to, int gender, const char *victim, con
 	} while (*from++);
 }
 
+
+/*mobjtype_t SL_DropWeapon(AActor* target)
```

Comment:
Not sure why this was added.  Please remove.

Reply:
Done.